app/assets/javascripts/room.js

```diff
@@ -50,6 +50,36 @@ $(document).on('turbolinks:load', function(){
       showDeleteRoom(this)
     })
 
+    // For keyboard users to be able to generate access code
```

Comment:
Same as for the other PR - can this be moved into a separate function?

Reply:
Sure, thanks for reviewing the codes, I have made the changes, please check it again when you have time